diff --git a/dist/client/client.mjs b/dist/client/client.mjs
index 3b2350cb110569a74966a9cd33638e4495add775..eaeb7e9b7a99e605430631268a5ef186394de58e 100644
--- a/dist/client/client.mjs
+++ b/dist/client/client.mjs
@@ -896,7 +896,7 @@ const debounceReload = (time) => {
 			timer = null;
 		}
 		timer = setTimeout(() => {
-			location.reload();
+			console.warn("VITE: PAGE REFRESH REQUIRED");;
 		}, time);
 	};
 };
@@ -933,7 +933,7 @@ async function handleMessage(payload) {
 		case "update":
 			await hmrClient.notifyListeners("vite:beforeUpdate", payload);
 			if (hasDocument) if (isFirstUpdate && hasErrorOverlay()) {
-				location.reload();
+				console.warn("VITE: PAGE REFRESH REQUIRED");;
 				return;
 			} else {
 				if (enableOverlay) clearErrorOverlay();
@@ -971,7 +971,7 @@ async function handleMessage(payload) {
 					const url = new URL(socket.url);
 					url.search = "";
 					await waitForSuccessfulPing(url.href);
-					location.reload();
+					console.warn("VITE: PAGE REFRESH REQUIRED");;
 				}
 			}
 			break;
